Unity_sdk_sample/Unity_sdk_sample/Assets/Viveport_SDK_Sample/Script/ViveportSDK_Sample_TopApi.cs

```diff
@@ -53,10 +53,9 @@ private void InitStatusHandler(int code)          // The callback of Api.init()
     {
         if (code == SUCCESS)
         {
-            Debug.Log("VIVEPORT init pass");
             if (onInitComplete != null)
             {
-                onInitComplete.Invoke(code, "<color=#009900>ViveportSDK Init is Complete !!</color>");
+                Api.QueryRuntimeMode(QueryRunTimeHandler);
```

Comment:
// Query runtime mode before using SDK API. If the mode mismatch, it will cause unexpected errors.